MarsQACompetitionTask/POMTests/POM.cs

```diff
@@ -0,0 +1,103 @@
+﻿using MarsQACompetitionTask.Pages;
+using MarsQACompetitionTask.Utilities;
+using NUnit.Framework;
+using OpenQA.Selenium;
+using OpenQA.Selenium.Chrome;
+//using OpenQA.Selenium.DevTools.V114.Profiler;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace MarsQACompetitionTask.POMTests
+{
+    [TestFixture]
+    public class POM : CommonDriver
```

Comment:
What is pom test? Can you please name the test class to what feature you are testing! If it's certification then class should be call CetificationTest and SkillTest etc please change this and put tests where they belong